src/commands/cmd_bit.cc

```diff
@@ -314,6 +322,12 @@ class CommandBitfield : public Commander {
       cmds_.push_back(cmd);
     }
 
+    if constexpr (ReadOnly) {
+      if (group.Good()) {
```

Comment:
Hmmm:

```
... // Parse as origin

    if constexpr (ReadOnly) {
      if (!readonly_) {
         return {Status::RedisParseErr, "BITFIELD_RO only supports the GET subcommand"};
      }
```

Would it be more straitforward?

Reply:
Yes, I was just struggling about this, but it would require an additional assignment to `read_only_`.

```cpp
read_only_ = false;
if constexp (ReadOnly) {
  break;
}
```

I'm not sure which one is better

Reply:
Hmm maybe we don't need to break? Just parsing it is ok for me, seems it should not be a bottleneck?

Reply:
ok, get your point